util/network-devp2p/src/lib.rs

```diff
@@ -65,12 +65,14 @@ pub use host::NetworkContext;
 pub use node_table::{MAX_NODES_IN_TABLE, NodeId, validate_node_url};
 pub use service::NetworkService;
 
+mod disk;
```

Comment:
I think this name is too generic. How about "enr_persistence" or something else indicating its use?

Reply:
Actually, the code there is not related to ENR as it defines the `DiskEntity` trait and an implementation for `parity_crypto::publickey::Secret`. Maybe just `persistence` then?

Reply:
I agree with David, `persistant_storage` or something seems better than disk.

Reply:
Renamed